spring-cloud-gcp-pubsub-stream-binder/src/main/java/com/google/cloud/spring/stream/binder/pubsub/provisioning/PubSubChannelProvisioner.java

```diff
@@ -79,7 +80,14 @@ public ConsumerDestination provisionConsumerDestination(String topicName, String
 
 		String subscriptionName;
 		Subscription subscription;
-		if (StringUtils.hasText(group)) {
+		if (StringUtils.hasText(properties.getExtension().getCustomSubscription())) {
```

Comment:
Can `properties.getExtension()` be null?

Reply:
I don't think so - [here](https://github.com/spring-cloud/spring-cloud-stream/blob/master/spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binding/BindingService.java#L106-L113) appears to be where the extended properties are created.